source/secure_tunneling.c

```diff
@@ -2535,7 +2589,7 @@ int aws_secure_tunnel_send_message(
         secure_tunnel->allocator, secure_tunnel, message_options, AWS_STOT_MESSAGE);
 
     if (message_op == NULL) {
-        return aws_last_error();
+        goto error;
```

Comment:
Not entirely sure why we need a specific `goto error` and `error:` if it's only being used once.